crates/sui-json-rpc/src/unit_tests/rpc_server_tests.rs

```diff
@@ -67,19 +67,25 @@ async fn test_public_transfer_object() -> Result<(), anyhow::Error> {
     let keystore = Keystore::from(FileBasedKeystore::new(&keystore_path)?);
     let tx = to_sender_signed_transaction(transaction_bytes.to_data()?, keystore.get_key(address)?);
     let (tx_bytes, sig_scheme, signature_bytes, pub_key) = tx.to_network_data_for_execution();
+    let tx_bytes1 = tx_bytes.clone();
```

Comment:
I think you can avoid the temp variable. Pass in the first use the .clone() and in the second without .clone()